orc8r/gateway/python/magma/directoryd/rpc_servicer.py

```diff
@@ -198,4 +222,17 @@ def GetAllDirectoryRecords(self, request, context):
                 directory_record.fields[identifier_key] = \
                     stored_record.identifiers[identifier_key]
 
+        self._print_grpc(response)
         return response
+
+    def _print_grpc(self, message):
+        if self._print_grpc_payload:
+            log_msg = "{} {}".format(message.DESCRIPTOR.full_name,
+                                     MessageToJson(message))
+            # add indentation
+            padding = 2 * ' '
+            log_msg =''.join( "{}{}".format(padding, line)
+                              for line in log_msg.splitlines(True))
+
+            log_msg = "GRPC message:\n{}".format(log_msg)
```

Comment:
log_msg = "GRPC: ... or log_msg = "directoryd message: ?

Reply:
See a sample of sessiond log. The log itself already include the name of the service 

Since the log contains some text lines and grpc messages i want to differentiate between them

Also this is in line with the other services that print grpc 

```
Feb 16 19:20:20 agw sessiond[30354]: I0216 19:20:20.857368 30397 GrpcMagmaUtils.cpp:38]
Feb 16 19:20:20 agw sessiond[30354]:   magma.lte.UpdateTunnelIdsRequest {
Feb 16 19:20:20 agw sessiond[30354]:       sid {                                                                                       Feb 16 19:20:20 agw sessiond[30354]:         id: "IMSI999991234567810"
Feb 16 19:20:20 agw sessiond[30354]:       }
Feb 16 19:20:20 agw sessiond[30354]:       bearer_id: 5                                                                                Feb 16 19:20:20 agw sessiond[30354]:       enb_teid: 419430400
Feb 16 19:20:20 agw sessiond[30354]:       agw_teid: 47
Feb 16 19:20:20 agw sessiond[30354]:   }

```